### PR TITLE
Remove remaining jQuery .css code

### PR DESCRIPTION
**Pull request by silverwind**
The linter missed these because they were set on a object. Tested and I also renamed those properties to add `$` indicating a jQuery selection.

### CODE REVIEW

File: web_src/js/features/imagediff.js [hunk 1]
```diff
@@ -220,38 +220,42 @@ export function initImageDiff() {
...
+        container.style.height = `${sizes.size2.height * factor + 2}px`;
 
-      const $range = $container.find("input[type='range']");
-      const onInput = () => sizes.image1.parent().css({
-        opacity: $range.val() / 100,
-      });
-      $range.on('input', onInput);
-      onInput();
+        // some inner elements are `position: absolute`, so the container's height must be large enough
+        // the "css(width, height)" is somewhat hacky and not easy to understand, it could be improved in the future
```
Comment (delvh) [inline web_src/js/features/imagediff.js]: That sentence no longer applies?
```suggestion
```

File: web_src/js/features/imagediff.js [hunk 2]
```diff
@@ -220,38 +220,42 @@ export function initImageDiff() {
...
+        overlayFrame.style.width = `${sizes.max.width * factor + 2}px`;
+        overlayFrame.style.height = `${sizes.max.height * factor + 2}px`;
+      }
+
+      const rangeInput = $container[0].querySelector('input[type="range"]');
```
Comment (delvh) [inline web_src/js/features/imagediff.js:250]: Huh?
Where is the container coming from?
I can see in the old code that it is equal to previously, but where is it declared?
It just seems to pop up out of nowhere.

Comment (silverwind) [reply web_src/js/features/imagediff.js:250]: It's a huge function, defined on line 72 `const $container = $(this);`

Comment (delvh) [reply web_src/js/features/imagediff.js:250]: I didn't expect a function definition this long **inside** another function.
Interesting.

**Review by delvh — APPROVED** [review]

File: web_src/js/features/imagediff.js (hunk 2 above)
Comment (silverwind) [reply web_src/js/features/imagediff.js:250]: Yes, this code is a huge pile of poop basically.

**Review by wxiaoguang — APPROVED** [review]

File: web_src/js/features/imagediff.js (hunk 1 above)
Comment (wxiaoguang) [reply web_src/js/features/imagediff.js]: It still applies

Comment (silverwind) [reply web_src/js/features/imagediff.js]: Anyone looking at that code knows it's hacky 😆

Comment (silverwind) [reply web_src/js/features/imagediff.js]: Oh and sorry about merging too soon.

**Pull request merged**